source/helpers/dom-utils.ts

```diff
@@ -107,3 +108,17 @@ export function removeTextInTextNode(node: Text | ChildNode, text: RegExp | stri
 	assertNodeContent(node, text);
 	node.textContent = node.textContent.replace(text, '');
 }
+
+export function getElementByAriaLabelledBy<T extends HTMLElement>(baseSelector: string, label: string): T {
```

Comment:
Cool new helper because GitHub developers don't know to create an HTML forms anymore